openpype/hooks/pre_new_console_apps.py

```diff
@@ -25,7 +25,11 @@ def execute(self):
         # Set `stdout` and `stderr` to None so new created console does not
         #   have redirected output to DEVNULL in build
         self.launch_context.kwargs.update({
```

Comment:
I suggest this modification
```suggestion
        for key, value in (
            ("stdout", None),
            ("stderr", None),
            ("creationflags", subprocess.CREATE_NEW_CONSOLE),
        ):
            if key in self.launch_context.kwargs:
                continue
            self.launch_context.kwargs[key] = value
```

Reply:
This is coming from dependent #5903. So will implement in that PR.